### PR TITLE
Implement new slider screen

### PR DESCRIPTION
**Pull request by MaikelStuivenberg**
Refactored a lot of variables from org to collectgroup to make it technically correct :) (because for example the organisation type is something different then collectgroup) 
Also implemented the updated slider screen

## Summary by CodeRabbit

- **New Features**
	- Introduced `CollectGroupDetailsCubit` to manage collect group details, replacing the previous organization details management.
	- Added a new factory constructor for `FunTopAppBar` to create a white-themed app bar.
	- Introduced `FunGive` widget for customizable circular graphics.

- **Improvements**
	- Streamlined various screens and widgets to focus on collect group details, enhancing user experience.
	- Updated state management across multiple components to reflect the new collect group context.

- **Chores**
	- Updated asset management to include new illustration assets.

### CODE REVIEW

File: lib/features/family/features/giving_flow/screens/choose_amount_slider_screen.dart [hunk 1]
```diff
@@ -51,84 +55,119 @@ class ChooseAmountSliderScreen extends StatelessWidget {
...
-                SliderWidget(state.amount, state.maxAmount),
-                const Spacer(),
-              ],
-            ),
+        return FunScaffold(
+          appBar: FunTopAppBar(
+            title: collectgroup.name,
+            leading: const GivtBackButtonFlat(),
+            actions: [
+              showActionIcon(collectgroup),
```
Comment (TammiLion) [inline lib/features/family/features/giving_flow/screens/choose_amount_slider_screen.dart:63]: Can we please follow flutter naming convention? this "show" stuff really irks me 😂 Especially for declarative stuff, opinion?

Comment (MaikelStuivenberg) [reply lib/features/family/features/giving_flow/screens/choose_amount_slider_screen.dart:63]: haha, it was used in different places in this way, so I'll just continue 😂

What would you prefer? 
just actionIcon() ? :)

File: lib/features/family/features/giving_flow/screens/choose_amount_slider_screen.dart [hunk 2]
```diff
@@ -51,84 +55,119 @@ class ChooseAmountSliderScreen extends StatelessWidget {
...
       },
     );
   }
 
-  Widget _getAppBarAction(FlowsState flow) {
-    if (flow.isQRCode || flow.isRecommendation) {
-      return const Padding(
-        padding: EdgeInsets.only(right: 16),
-        child: Wallet(),
+  Widget showActionIcon(
```
[the remark this answers is not in the document]
Comment (MaikelStuivenberg) [reply lib/features/family/features/giving_flow/screens/choose_amount_slider_screen.dart:112]: Yeah i'm also not sure. It's really specific logic for this page at the moment.. 
Also for parents giving and goal giving its already different

**Review by TammiLion — APPROVED** [review]
Approved but please take a look at the naming!

File: lib/features/family/features/giving_flow/screens/choose_amount_slider_screen.dart (hunk 1 above)
Comment (TammiLion) [reply lib/features/family/features/giving_flow/screens/choose_amount_slider_screen.dart:63]: actionIcon definitely!

Comment (TammiLion) [reply lib/features/family/features/giving_flow/screens/choose_amount_slider_screen.dart:63]: For me show just makes more sense as an "action". Like showbottomSheet, showPopUp etc

Comment (MaikelStuivenberg) [reply lib/features/family/features/giving_flow/screens/choose_amount_slider_screen.dart]: Thanks! it's changed

**Pull request merged**